SingularityUI/app/components/requestDetail/header/RequestAlerts.jsx

```diff
@@ -74,14 +73,14 @@ const RequestAlerts = ({requestId, requestAPI, bounces, activeTasksForRequest})
           }
           maybeDeployProgress = (
             <span>
-              Finished deploying {targetActiveInstances} of {instances} total instances, {nextDeployStepRemark}
+              Finished deploying {targetActiveInstances} total instances, {nextDeployStepRemark}
             </span>
           );
         } else {
           maybeDeployProgress = (
             <span>
               {
-                `Trying to deploy ${targetActiveInstances} of ${instances}
```

Comment:
The problem here is that `of` didn't make sense. `targetActiveInstances` is not a fraction of `instances` in all cases. Instead, if the deploy is successful, it will override `instances`. If we want to continue mentioning `instances`, better wording may be `Trying to deploy ${targetActiveInstances} (instead of ${instances}) instances.`